### PR TITLE
Feature/front end wt

### PR DESCRIPTION
**Pull request by pcrespov**
Added a demo of web UI with wt

### CODE REVIEW

File: demos/frontend-wt/.gitattributes [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+# Set the merge driver for project and solution files
+#
+# Merging from the command prompt will add diff markers to the files if there
+# are conflicts (Merging from VS is not affected by the settings below, in VS
+# the diff markers are never inserted). Diff markers may cause the following 
+# file extensions to fail to load in VS. An alternative would be to treat
+# these files as binary and thus will always conflict and require user
+# intervention with every merge. To do so, just uncomment the entries below
+###############################################################################
+#*.sln       merge=binary
```
Comment (sbenkler) [inline demos/frontend-wt/.gitattributes]: Are we sure, to merge as binary ... do we still get nice diffs? Or simply: the file has changed?
I agree, that VS is sometimes too smart to rearrange but many times, it does the job.

-> is it still needed with cmake?

File: demos/frontend-wt/.gitignore [hunk 2]
```diff
@@ -0,0 +1,265 @@
+## Ignore Visual Studio temporary files, build results, and
+## files generated by popular Visual Studio add-ons.
+
+# User-specific files
+*.suo
+*.user
+*.userosscache
```
Comment (sbenkler) [inline demos/frontend-wt/.gitignore]: We should favor out-source-builds.

Comment (pcrespov) [reply demos/frontend-wt/.gitignore]: All these files were automatically generated by msvs

File: demos/frontend-wt/.gitattributes (hunk 1 above)
Comment (pcrespov) [reply demos/frontend-wt/.gitattributes]: The original version by EC was done with VS, then i created the makefiles. 
I agree with you. In the future we will not include any of this

**Pull request merged**